common/eth2/src/lib.rs

```diff
@@ -1046,6 +1046,23 @@ impl BeaconNodeHttpClient {
         Ok(())
     }
 
+    // GET builder/states/{state_id}/expected_withdrawals
+    pub async fn get_expected_withdrawals<T:EthSpec>(
+        &self,
+        state_id: &StateId,
+    ) -> Result<GenericResponse<NextWithdrawalsResponse<VariableList<Withdrawal,T>>>, Error> {
```

Comment:
Looks like this isn't formatted with `rustfmt` which you can fix with `cargo fmt --all`.